src/app/pages/guides/cfar-offer-banner-large/cfar-offer-banner-large.component.html

```diff
@@ -9,7 +9,7 @@
         <mat-tab label="Overview">
             <div class="p-16">
                 <hopper-cfar-offer-banner-large
-                    [env]="'development'"
+                    env="development"
```

Comment:
if you pass **development** without ' inside the ", it needs to be a variable but in your case it seems that it's a hardcoded string. 

If it's a string, you can do like this : 
`env="'development'"`

Reply:
looks like` env="development"`  without square brackets and `[env]="'development'"` does the same thing.  so kept the one with least characters :P